packages/client/src/BrubeckNode.ts

```diff
@@ -236,4 +236,26 @@ export default class BrubeckNode implements Context {
             this.debug('closeProxyConnectionOnStream << %o', streamPartId, nodeId)
         }
     }
+
+    async openSubscribeProxyConnectionOnStreamPart(streamPartId: StreamPartID, nodeId: string): Promise<void> {
+        try {
+            if (!this.cachedNode || !this.startNodeComplete) {
```

Comment:
I'm thinking this `!this.cachedNode || !this.startNodeComplete` should probably become a method e.g. 
```ts
isStarting(): boolean {
  return !this.cachedNode || !this.startNodeComplete
}
```
Low priority though.